caliban/platform/cloud/core.py

```diff
@@ -429,7 +431,19 @@ def _job_specs(
 
   """
   for idx, m in enumerate(experiments, 1):
-    args = ce.experiment_to_args(m.kwargs, m.args)
+    wrapper_args = [f'--caliban_env="{k}={v}"' for k, v in env.items()]
```

Comment:
can we have this not leak directly into `cloud.core`, and configure this outside?